src/assets/movieArray.ts

```diff
@@ -0,0 +1,174 @@
+import { Movie } from 'src/app/main/models/movie.model';
```

Comment:
If file contains mocks, then add a mock to the filename, for example movieArray.ts -> movieArray.mock.ts. And I recommend creating a separate folder for mocks